roles/setup_pgbackrest/tasks/configure_pgbackrest.yml

```diff
@@ -21,6 +21,15 @@
   when:
     - use_hostname|bool
 
+- name: Set _pgbackrest_host when using hostname
```

Comment:
`name` does not reflect what the task is doing (wrong var. name)

Reply:
I have update this task `name` and added the `_standby_index_var` variable to the `Reset local variables` task.